tests/deposit-redeem-flow.ts

```diff
@@ -36,6 +35,16 @@ describe("deposit-redeem-flow", () => {
 
   let userA: Keypair;
 
+  // Pyth USDC - Devnet
+  const pyth_usdc_devnet = new PublicKey(
```

Comment:
let's adhere to javascript naming conventions, i.e. `pyth_usdc_devnet` -> `pythUsdcDevnet`. same for switchboard.

Reply:
also, we might want a more robust way to provide the oracle pubkeys. e.g. on localnet, use `mockOracle` for pyth and use pubkey directly on non-localnet networks.

thinking that straight-up removing `mockOracle` will cause Pyth calls to fail on localnet.

Reply:
Yeah, a goal of mine is to create testing scripts for each cluster. I'll look more into this once I can properly mock Switchboard on localnet.